client/packages/system/src/Stock/DetailView/AppBarButtons.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react';
+import {
+  AppBarButtonsPortal,
+  ButtonWithIcon,
+  Grid,
+  useTranslation,
+  StockIcon,
+  BarChartIcon,
+} from '@openmsupply-client/common';
+
+interface AppBarButtonProps {
+  //
+}
+
+export const AppBarButtonsComponent: FC<AppBarButtonProps> = ({}) => {
+  // const { OpenButton } = useDetailPanel();
+  const t = useTranslation('inventory');
+
+  return (
+    <AppBarButtonsPortal>
+      <Grid container gap={1}>
+        <ButtonWithIcon
+          // disabled={isDisabled}
+          label={t('button.repack')}
+          Icon={<StockIcon />}
+          // TO-DO: Add repack modal
+          onClick={() => {}}
+        />
+        <ButtonWithIcon
+          // disabled={isDisabled}
+          label={t('button.adjust')}
+          Icon={<BarChartIcon />}
```

Comment:
Not sure about this icon. There was nothing in the icon library that seemed totally appropriate -- can add a new one if we think it needs it?

Reply:
Oof yeah that one looks weird... mayybe RefreshIcon? Though [this](https://mui.com/material-ui/material-icons/?query=arrow+up+down&selected=SwapVert) might be nicer?